primitives/src/lib.rs

```diff
@@ -110,12 +110,14 @@ pub const RING_HARD_CAP: Balance = 10_000_000_000 * COIN;
 pub const TOTAL_POWER: Power = 1_000_000_000;
 
 /// Deposit calculator for Darwinia.
+/// 5000 RING for the base fee, 102.4 RING/MB.
 pub const fn darwinia_deposit(items: u32, bytes: u32) -> Balance {
-	items as Balance * 20 * MILLI + (bytes as Balance) * 100 * NANO
+	items as Balance * 5000 * COIN + (bytes as Balance) * 100 * MILLI
```

Comment:
Not sure about the definition of items, vote? proposal? 

OK for proposal, but maybe too large for vote.

Reply:
```suggestion
	items as Balance * 100 * COIN + (bytes as Balance) * 10 * MICRO
```

This is the cost for storage, and should add gov cost separately if necessary.